chandra.time/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set data = load_setup_py_data() %}
```

Comment:
It looks like conda-build 2.1.15 works fine with this (and ignores the the bits that don't JSON serialize).  conda-build 2.1.16 and conda-build 3.0.6 seem to break on the same recipe, however.  Oy.